test/cast/cast_receiver_integration.js

```diff
@@ -205,7 +205,7 @@ filterDescribe('CastReceiver', castReceiverIntegrationSupport, () => {
         // Check that the update message is of a reasonable size. From previous
         // testing we found that the socket would silently reject data that got
         // too big. 6KB is safely below the limit.
-        expect(message.length).toBeLessThan(6000);
+        expect(message.length).toBeLessThan(6077);
```

Comment:
it seems like with the new config, one of the cast messages, which includes the config, ends up being a bit larger than 6000. Not sure how we want to handle this, as this means that this number will go up with every new config option.

Additionally, even if we don't have config options for this, a few config options will put us over this threshold, as the current message size sits at `5941`. @joeyparrish @avelad

Reply:
oops, would need to be 6078 or be changed to `toBeLessThanOrEqual`. But I'll wait on that change until we have a decision on how we want to proceed here.

Reply:
Just bump it to 7000.  That whole subsystem is going to be removed later.

Reply:
cool, just didn't want to bump it without someone who knows more about it looking at it. Thanks!